libs/barista-components/experimental/datepicker/src/timepicker.ts

```diff
@@ -53,6 +53,46 @@ export class DtTimepicker {
   @Input()
   minute: number | null;
 
+  /** The minimum selectable hour. */
+  @Input()
+  get minHour(): number | null {
+    return this._minHour;
+  }
+  set minHour(value: number | null) {
+    this._minHour = value;
+  }
+  private _minHour: number | null = null;
+
+  /** The minimum selectable minute. */
+  @Input()
+  get minMinute(): number | null {
+    return this._minMinute;
+  }
+  set minMinute(value: number | null) {
+    this._minMinute = value;
+  }
+  private _minMinute: number | null = null;
+
+  /** The maximum selectable hour. */
+  @Input()
+  get maxHour(): number | null {
+    return this._maxHour;
+  }
+  set maxHour(value: number | null) {
+    this._maxHour = value;
+  }
+  private _maxHour: number | null = null;
+
+  /** The maximum selectable minute. */
+  @Input()
+  get maxMinute(): number | null {
+    return this._maxMinute;
+  }
+  set maxMinute(value: number | null) {
+    this._maxMinute = value;
+  }
+  private _maxMinute: number | null = null;
```

Comment:
It does not seem that there is a lot of logic going on, is there a reason why this input has getter and setters instead of `@Input() maxMinute: string | number | null = null`